examples/mcp-integrations/README.md

```
@@ -0,0 +1,135 @@
+```bash
+killall "Claude" && open -a "Claude"
+```
+
+### 3. Run Health Check
+
+```bash
+node ../../tools/mcp/community-mcp-health-monitor.cjs
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)